cmd/extension/run.go

```diff
@@ -0,0 +1,40 @@
+package extension
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/odpf/optimus/extension"
+	"github.com/odpf/optimus/extension/model"
+)
+
+func generateRunCommands(reservedCommandNames []string) []*cobra.Command {
+	manifester := extension.NewDefaultManifester()
+	manifest, err := manifester.Load(model.ExtensionDir)
+	if err != nil {
+		panic(err)
```

Comment:
Will this panic show useful error message to the cli user ?

Reply:
updated by adding more useful message to help the user